### ISSUE TITLE
Add support for climbing areas and OSM relations

### ISSUE THREAD

**Issue by xyz32** [xyz-relativity/ClimbTheWorld-Android#7]
Usually climbing crags are part of a climbing area.

overpass turbo:
`[out:json][timeout:60];
	area[type=boundary]["ISO3166-1"="CA"]->.searchArea;
	(
	  node["sport"~"^climbing$|^climbing[:space:;]|[:space:;]climbing[:space:;]|[:space:;]climbing$"](area.searchArea);
      way["sport"~"^climbing$|^climbing[:space:;]|[:space:;]climbing[:space:;]|[:space:;]climbing$"](area.searchArea);
      >;
	);

	out body meta;
	rel(bn);
	out body meta;
	rel(br);
	out body meta;`

**Comment by olbotta** [comment]
i agree, since from what i can understand from the [wiki](https://wiki.openstreetmap.org/wiki/Tag:sport%3Dclimbing) if you map all the routes they should be grouped into a relation, and in this case there would be no specific POI to represent the climbing crag. currently climb the world only recognises points and not relations.

**Comment by xyz32** [comment]
That may have been me who put that info into the wiki. At this time I am still trying to get more familiar with OSM relations to see how good of a fit for this they are.
The only concern I am having right now is the climbing areas, which would result in being mapped as a relation containing all the crag relations. A bit abstract to my liking, but maybe I should start implementing it, and it may work.

**Comment by alensiljak** [comment]
This would be great, to match the criteria specified by the wiki, and implemented by https://openclimbing.org/
